### DISCUSSION TITLE
SiparisDetail  Create Error

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5466] [Help]
![image](https://user-images.githubusercontent.com/24252615/107495002-8ed20500-6ba0-11eb-9e35-c6990b79f8e3.png)
I'm trying to make an order page as in Nortwind, but SiparisDetail creating error occurs when I create it. Where could I be doing wrong?

**Comment by VictorTomaili** [comment]
You need provide english error message and source code of your dialog etc.
